src/google/protobuf/compiler/command_line_interface.cc

```diff
@@ -188,8 +188,25 @@ bool TryCreateParentDirectory(const string& prefix, const string& filename) {
 // Get the absolute path of this protoc binary.
 bool GetProtocAbsolutePath(string* path) {
 #ifdef _WIN32
-  char buffer[MAX_PATH];
-  int len = GetModuleFileName(NULL, buffer, MAX_PATH);
+  wchar_t wide_buffer[MAX_PATH];
+  // Note: the returned length does not include terminating null character.
+  int len = GetModuleFileNameW(NULL, wide_buffer, MAX_PATH);
```

Comment:
As far as I know, there are people want to use GetModuleFileNameA here.
It's better to figure out the macro to distinguish the situation to use different GetModueFileName.